adapters/repos/db/bm25f_test.go

```diff
@@ -264,8 +264,8 @@ func TestBM25FDifferentParamsJourney(t *testing.T) {
 	}
 
 	// Check scores
-	require.Equal(t, float32(0.056813046), res[0].Score())
-	require.Equal(t, float32(0.054633126), res[1].Score())
+	require.Equal(t, float32(2.805209), res[0].Score())
+	require.Equal(t, float32(0.06341988), res[1].Score())
```

Comment:
Sorry about that. Might have been me. Had a super weird merge conflict about this on a branch. Thought I managed to fix it without messing everything up. Looks like I didn't. Thanks for fixing it again.

Reply:
The numbers started bouncing around with each test run, so I lowered the precision.  Might be something we should look into later